xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -287,6 +287,10 @@ bool CGUIWindowMusicNav::GetDirectory(const std::string &strDirectory, CFileItem
       OnRetrieveMusicInfo(items);
   }
 
+  //Navigating music files so default content to "songs" unless in sources folder.
+  //This content allows view type to include media info so that file tag data can be displayed
+  if (!URIUtils::IsSourcesPath(strDirectory))
```

Comment:
FYI, this is also causing the scrape issues @MartijnKaijser pointed out on the forums. Due to wrong content set on the items, the `MusicInfoScanner` tries to scrape artist information with the album scraper.